brainscore/benchmarks/_properties_common.py

```diff
@@ -0,0 +1,422 @@
+import numpy as np
+
+from brainio_base.assemblies import DataAssembly
+from brainscore import get_stimulus_set
+from brainscore.benchmarks import BenchmarkBase, ceil_score
+from brainscore.benchmarks.screen import place_on_screen
+from brainscore.model_interface import BrainModel
+from result_caching import store
+
+BLANK_STIM_NAME = 'dicarlo.Marques2020_blank'
+RF_STIM_NAME = 'dicarlo.Marques2020_receptive_field'
+ORIENTATION_STIM_NAME = 'dicarlo.Marques2020_orientation'
+
+RF_NUMBER_OF_TRIALS = 10
+ORIENTATION_NUMBER_OF_TRIALS = 20
+RF_THRSH = 0.05
+RF_DELTA = 0.15
+MEDIAN_MAX_RESP = {'V1': 33.8}
+MEDIAN_SPONTANEOUS = {'V1': 0.82}
+RESP_THRESH = {'V1': 5}
+LOW_INTERVAL_MAX_RESP = {'V1': 11.14}
+HIGH_INTERVAL_MAX_RESP = {'V1': 86.27}
+LOW_INTERVAL_PERCENTILE = 10
+HIGH_INTERVAL_PERCENTILE = 90
+
+
+class PropertiesBenchmark(BenchmarkBase):
```

Comment:
However the LGN and V2 properties benchmarks that I also started working on use this same class. This is not specific for V1.

Reply:
Right so maybe `NeuralPropertiesBenchmark`? I'm bringing it up because when Paul wanted to implement the topography benchmark, it got confusing whether he should sub-class the `PropertiesBenchmark` class, but it doesn't apply there